consensus/src/payload_manager/quorum_store_payload_manager.rs

```diff
@@ -289,6 +290,47 @@ impl TPayloadManager for QuorumStorePayloadManager {
         };
     }
 
+    async fn check_denied_inline_transactions(
```

Comment:
this is not async function?

Reply:
Aah, thanks, removed! (It was `async` when I was doing silly stuff 😆)